scripts/minikube.sh

```diff
@@ -54,10 +54,6 @@ function install_minikube() {
         local mk_version version
         read -ra mk_version <<<"$(${minikube} version)"
         version=${mk_version[2]}
```

Comment:
If the requested version is `latest`, this check can be skipped. In other cases, it should be kept.

Reply:
@nixpanic have you looked at the next new commit, where the commit tries to update the minikube if the latest version exists.

Thanks!

Reply:
I somehow missed that, thanks!